.github/workflows/linters.yml

```diff
@@ -25,7 +29,7 @@ jobs:
       - name: Get changed C++ files
         if: success() || failure()
         id: changed-cxx-files
-        uses: tj-actions/changed-files@v46
+        uses: tj-actions/changed-files@ed68ef82c095e0d48ec87eccea555d944a631a4c # v46.0.5
```

Comment:
Would leaving these comments in start to be confusing as tags are updated? Or is dependabot smart enough to update?

Reply:
Dependabot is, fortunately, smart enough to update the comments 😁